src/streaming/rules/abr/L2ARule.js

```diff
@@ -388,56 +389,65 @@ function L2ARule(config) {
                 let throughputMeasureTime = dashMetrics.getCurrentHttpRequest(mediaType).trace.reduce((a, b) => a + b.d, 0);
                 const downloadBytes = dashMetrics.getCurrentHttpRequest(mediaType).trace.reduce((a, b) => a + b.b[0], 0);
                 let lastthroughput = Math.round((8 * downloadBytes) / throughputMeasureTime); // bits/ms = kbits/s
+                let currentHttpRequest = dashMetrics.getCurrentHttpRequest(mediaType);
 
                 if (lastthroughput < 1) {
                     lastthroughput = 1;
                 }//To avoid division with 0 (avoid infinity) in case of an absolute network outage
 
-                let V = l2AState.lastSegmentDurationS;
-                let sign = 1;
+                if (currentHttpRequest.url == l2AState.lastSegmentUrl ||
```

Comment:
Probably not relevant for low latency streaming but `SegmentBase` with `rangeRequests` should always use the same URL. I suggest we add a comment here to highlight this.

Reply:
Did you have some suggested text?

Reply:
Something like: "Note that for `SegmentBase` addressing the request url does not change. As this is not relevant for low latency streaming at this point the check below is sufficient"